### PR TITLE
Setup mobile first

### PR DESCRIPTION
**Pull request by Adetunbosun**
In this PR I
- Use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- Build
 the toolbar (or header):
    - the headline section (right after the header):
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+## Built With
+
+- HTML
+- CSS
+- VSCODE
+
+## Live Demo (if available)
+
+> Loading...
```
Comment (emmiiorji) [inline README.md]: Kindly remove the "if available" as it is only a note to the author.

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+### Deployment
+N/A
+
+## Authors
+
+👤 **Olusola**
+
+- GitHub: [@Adetunbosun](https://github.com/Adetunbosun)
+- Twitter: [@olusola](https://twitter.com/twitterhandle)
+- LinkedIn: [Olusola](https://linkedin.com/in/linkedinhandle)
```
Comment (emmiiorji) [inline README.md]: Please, replace the `twitterhandle` and `linkedinhandle` in the URLs with your Twitter and LinkedIn handles respectively. This is so as not to make broken links.

File: README.md [hunk 3]
```diff
@@ -1 +1,63 @@
...
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (emmiiorji) [inline README.md]: In order to make your documentation neater, kindly remove this footnote as it is only a note to the author of the project (you).

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Adetunbosun , 

Good job so far!
There are just a few issues that you still need to work on to go to the next project but you are almost there!

Just-a-little-bit


## Good Highlights ✅

- Good commit messages
- Linter checks are passed
- Design is pretty and close to the chosen template

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @Adetunbosun  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**